libs/journeys/ui/src/components/Card/ContainedCover/ContainedCover.tsx

```diff
@@ -73,6 +73,8 @@ export function ContainedCover({
     }
   }, [imageBlock, theme, videoBlock, backgroundBlur])
 
+  const videoImage = videoBlock?.image ?? videoBlock?.video?.image
```

Comment:
Wonder if this is worth making deliberate.

```suggestion
  const videoImage = videoBlock?.source === VideoBlockSource.internal ? videoBlock?.video?.image : videoBlock?.image
```

Reply:
same also for the video component.